app/src/main/kotlin/com/intive/picover/parties/viewmodel/PartiesViewModel.kt

```diff
@@ -0,0 +1,26 @@
+package com.intive.picover.parties.viewmodel
+
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import com.intive.picover.common.viewmodel.LifecycleViewModel
+import com.intive.picover.parties.repository.MockedPartiesRepository
+import com.intive.picover.parties.state.PartiesState
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+
+@HiltViewModel
+class PartiesViewModel @Inject constructor(
+	private val partiesRepository: MockedPartiesRepository,
+) : LifecycleViewModel() {
+
+	val parties: MutableState<PartiesState> = mutableStateOf(PartiesState.Loading)
+
+	override fun onFirstCreate() {
+		super.onFirstCreate()
```

Comment:
super.onFirstCreate() can be omitted here as well.

Reply:
cc @MarekMacko

Reply:
Done